Atomic.md

```diff
@@ -0,0 +1,618 @@
+Regardless of how many accounts' transactions are included in an `Atomic` transaction, all accounts need to sign the collection of transactions.
+
+#### 5.1.1. Single Account
+In the single account case, this is obvious; the single account must approve all of the transactions it is submitting. No other accounts are involved, so this is a pretty straightforward case.
+
+#### 5.1.2. Multi Account
+The multi-account case is a bit more complicated and is best illustrated with an example. Let's say Alice and Bob are conducting a trustless atomic swap via a multi-account `Atomic`, with Alice providing 1000 XRP and Bob providing 1000 USD. Bob is going to submit the `Atomic` transaction, so Alice must provide her part of the swap to him.
+
+If Alice provides a fully autofilled and signed transaction to Bob, Bob could submit Alice's transaction on the ledger without submitting his and receive the 1000 XRP without losing his 1000 USD. Therefore, the inner transactions must be unsigned. 
+
+If Alice just signs her part of the `Atomic` transaction, Bob could modify his transaction to only provide 1 USD instead, thereby getting his 1000 XRP at a much cheaper rate. Therefore, the entire `Atomic` transaction (and all its inner transactions) must be signed by all parties.
+
+## 6. Examples
+
+### 6.1. One Account
+
+In this example, the user is creating an offer while trading on a DEX UI, and the second transaction is a platform fee.
+
+#### 6.1.1. Sample Transaction
+
+<details open>
+<summary>
+
+The inner transactions are not signed, and the `AtomicSigners` field is not needed on the outer transaction, since there is only one account involved.
+</summary>
+
+```typescript
+{
+  TransactionType: "Atomic",
+  Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
+  Flags: "1",
+  TxnIDs: [
+    "7EB435C800D7DC10EAB2ADFDE02EE5667C0A63AA467F26F90FD4CBCD6903E15E",
+    "EAE6B33078075A7BA958434691B896CCA4F532D618438DE6DDC7E3FB7A4A0AAB"
+  ],
+  RawTransactions: [
+    {
+      RawTransaction: {
+        TransactionType: "OfferCreate",
+        Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
+        TakerGets: "6000000",
+        TakerPays: {
+          currency: "GKO",
+          issuer: "ruazs5h1qEsqpke88pcqnaseXdm6od2xc",
+          value: "2"
+        },
+        AtomicTxn: {
+          Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
+          OuterSequence: 3,
+          BatchIndex: 0
+        },
+        Sequence: 0,
+        Fee: "0",
+        SigningPubKey: "",
+        TxnSignature: ""
+      }
+    },
+    {
+      RawTransaction: {
+        TransactionType: "Payment",
+        Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
+        Destination: "rDEXfrontEnd23E44wKL3S6dj9FaXv",
+        Amount: "1000",
+        AtomicTxn: {
+          Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
+          OuterSequence: 3,
+          BatchIndex: 1
+        },
+        Sequence: 0,
+        Fee: "0",
+        SigningPubKey: "",
+        TxnSignature: ""
+      }
+    }
+  ],
```

Comment:
Is it necessary to have the multiple levels of nesting like this? Could the list of transactions more simply be specified as:
```suggestion
  RawTransactions: [
    {
      TransactionType: "OfferCreate",
      Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
      TakerGets: "6000000",
      TakerPays: {
        currency: "GKO",
        issuer: "ruazs5h1qEsqpke88pcqnaseXdm6od2xc",
        value: "2"
      },
      AtomicTxn: {
        Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
        OuterSequence: 3,
        BatchIndex: 0
      },
      Sequence: 0,
      Fee: "0",
      SigningPubKey: "",
      TxnSignature: ""
    },
    {
      TransactionType: "Payment",
      Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
      Destination: "rDEXfrontEnd23E44wKL3S6dj9FaXv",
      Amount: "1000",
      AtomicTxn: {
        Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
        OuterSequence: 3,
        BatchIndex: 1
      },
      Sequence: 0,
      Fee: "0",
      SigningPubKey: "",
      TxnSignature: ""
    }
  ],
```

Reply:
I thought that was an unfortunate necessity of the serialization mechanism - that STArrays could only contain STObjects.

Reply:
You may be right about that. I don't remember offhand.

Reply:
I'd love to make that change if it's possible 😅  Let me know if you come across any information about it.